packages/polkadart_scale_codec/test/test_types/codec_compact_test.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:polkadart_scale_codec/polkadart_scale_codec.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('encode compact integers to hex', () {
+    test('Should return correct encoded data when value is zero', () {
+      final value = 0.toBigInt;
```

Comment:
nit: BigInt has an API for 0, 1 and 2 (chicken count lol)
```suggestion
      final value = BigInt.zero;
```